app/src/main/java/helium314/keyboard/settings/Icons.kt

```diff
@@ -18,7 +20,10 @@ import helium314.keyboard.latin.R
 
 @Composable
 fun NextScreenIcon() {
-    Icon(painterResource(R.drawable.ic_arrow_left), null, Modifier.scale(-1f, 1f))
+    Icon(
+        painterResource(R.drawable.ic_arrow_left), null,
+        if (Resources.getSystem().configuration.layoutDirection == View.LAYOUT_DIRECTION_LTR) Modifier.scale(-1f, 1f) else Modifier
```

Comment:
Does it work the same if you use compose? That would be `LocalLayoutDirection.current == LayoutDirection.Ltr`.